eslint.config.cjs

```diff
@@ -58,6 +58,20 @@ module.exports = [{
         },
     },
 
+    rules: {
+        "no-unused-vars": 0,
+    },
+}, {
+    files: ["example-apps/minimal-esm/*"],
+
+    languageOptions: {
+        ecmaVersion: 2025,
+        globals: {
+            ...globals.mocha,
+        },
+        sourceType: "module",
+    },
```

Comment:
I had to add this to keep the precommit hook from failing when it runs eslint, since eslint support for `import` attributes (`with { type: "json" }`) was added recently in https://github.com/eslint/eslint/pull/19076. The `typescript-esm` example app did not trigger these errors for some reason.